frontend/router/paths.ts

```diff
@@ -44,8 +43,8 @@ export default {
   EDIT_LABEL: (labelId: number): string => {
     return `${URL_PREFIX}/labels/${labelId}`;
   },
-  EDIT_QUERY: (query: IQuery): string => {
```

Comment:
Changed parameter to queryId for type number since it's being used in various places with various query interfaces that don't quite match but queryid is universal